accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/configreader/YamlConfigReader.java

```diff
@@ -467,5 +468,17 @@ protected String getMapValueAsString(
         }
         return "";
     }
+    
+    /**
+     * Normalize paths by removing superfluous "/" at the end to prevent unnecessary change detection
+     * @param path
+     * @return the normalized path
+     */
+    private static String normalizePath(String path) {
```

Comment:
We could just use `StringUtils.removeEnd(path, "/")` here (works also correct for `null` or values with trailing `/`)